src/PeriodEnd/CoInvestedPayments/DeployCoInvestedPayments/sql/Migration Scripts/1_CoInvestedPayments.deds.Add_Indexes.sql

```diff
@@ -3,7 +3,7 @@ JOIN sys.objects t ON i.object_id = t.object_id
 WHERE t.name = 'Collection_Period_Mapping'
 AND i.name = 'IX_PeriodMapping_YearOfCollection')
 BEGIN
-	CREATE INDEX IX_PeriodMapping_YearOfCollection ON [dbo].[Collection_Period_Mapping] (YearOfCollection);
+	CREATE INDEX IX_PeriodMapping_YearOfCollection ON [dbo].[Collection_Period_Mapping] (Collection_Year);
```

Comment:
Given the reason these changes didn't cause problems in our environments, can we change this to a drop and create?